.github/pull_request_template.md

```diff
@@ -0,0 +1,33 @@
+<!-- Thanks for contributing! -->
+## Summary
+<!-- What conceptually is this PR introducing? If context is already provided from the JIRA ticket, still place it in the
+Pull Request as you should not make the reviewer do digging for a basic summary. -->
+
+## Changes in this PR
+<!-- What changes did you make to the code? What new APIs (public or private) were added, removed, or edited to generate
+the desired outcome explained in the above summary? -->
+
+## Test Plan
+<!-- How did you test the code? If you added unit tests, you can say that. If you didn’t introduce unit tests, explain why.
+All code should be tested in some way – so please list what your validation strategy was. -->
+
+### Screenshots (optional)
+<!-- Usually a great supplement to a test plan, especially if this requires local testing. -->
+
+## Checklist
+<!-- Do not delete the items provided on this checklist. -->
+### Checklist for Author
+- [ ] Did you update the changelog (if necessary)?
+- [ ] Is the intention of the code captured in relevant tests?
+- [ ] If there are new TODOs, has a related JIRA ticket been created?
+
+### Checklist for Reviewer {@primary_reviewer}
+- [ ] Does the title of the PR reference a JIRA Ticket?
+- [ ] Do you fully understand the implementation? (Would you be comfortable explaining how this code works to someone else?)
+- [ ] Have you checked for spelling & grammar errors?
+- [ ] Is all relevant documentation (README or docstring) updated?
+
+## Focus Areas for Reviewer
+<!-- List any complex portion of code you believe needs additional scrutiny and explain why. -->
```

Comment:
This section seems a bit redundant. This is what one does when commenting on lines of the PR. Are you suggesting that these complex portions be added to the PR description?

Reply:
I understand now. I also like the idea of making it optional. I would leave as-is other than that.

Reply:
Its marked as optional now.

Reply:
@sleepyStick yes, that's the intent of the section, and having it as explicitly optional makes sense as well.